Tests/Routing/Loader/RestYamlCollectionLoaderTest.php

```diff
@@ -184,6 +214,36 @@ public function testControllerAsServiceWithClassName()
     }
 
     /**
+     * Test that YAML collection with named prefixes gets parsed correctly with inheritance.
+     */
+    public function testNamedPrefixedBaseReportsFixture()
+    {
+        $collection = $this->loadFromYamlCollectionFixture('base_named_prefixed_reports_collection.yml');
+        $etalonRoutes = $this->loadEtalonRoutesInfo('base_named_prefixed_reports_collection.yml');
+
+        foreach ($etalonRoutes as $name => $params) {
+            $route = $collection->get($name);
+            $methods = $route->getMethods();
+
+            $this->assertNotNull($route, $name);
+            $this->assertEquals($params['path'], $route->getPath(), $name);
+            $this->assertEquals($params['method'], $methods[0], $name);
+            $this->assertContains($params['controller'], $route->getDefault('_controller'), $name);
+        }
+
+        $name = 'api_get_billing_payments';
+        $this->assertArrayNotHasKey($name, $etalonRoutes);
+    }
+
+    public function testRoutesWithPattern()
```

Comment:
this test for [this](https://github.com/indragunawan/FOSRestBundle/blob/f2a557ac36284516c319e7925546907a6ab6eeee/Routing/Loader/RestYamlCollectionLoader.php#L126-L130)